modules/features/player/src/main/java/au/com/shiftyjelly/pocketcasts/player/view/ShareFragment.kt

```diff
@@ -4,124 +4,264 @@ import android.os.Bundle
+    onShareCurrentPosition: () -> Unit,
+    onShareOpenFileIn: () -> Unit,
+) {
+    Surface(
+        modifier = Modifier
+            .fillMaxWidth(),
+        color = backgroundColor
+    ) {
+        Column(
+            modifier = Modifier
+                .padding(bottom = 24.dp)
+        ) {
+            Pill(
+                modifier = Modifier
+                    .align(Alignment.CenterHorizontally)
+            )
+            if (podcast != null) {
+                ShareButton(
+                    modifier = Modifier
+                        .padding(top = 20.dp),
+                    label = IR.string.share_podcast,
+                    onClick = onSharePodcast
+                )
             }
-            close()
-        }
-        binding.buttonOpenFileIn.setOnClickListener {
-            if (podcast != null && episode is PodcastEpisode) {
-                SharePodcastHelper(
-                    podcast,
-                    episode,
-                    episode.playedUpTo,
-                    requireContext(),
-                    ShareType.EPISODE_FILE,
-                    SourceView.PLAYER,
-                    analyticsTracker
-                ).sendFile()
+            Divider(
+                color = Color.LightGray.copy(0.3f),
+                thickness = 1.dp,
+            )
+            Divider(thickness = 1.dp)
+            if (podcast != null && episode is BaseEpisode) {
+                ShareButton(
+                    label = IR.string.podcast_share_episode,
+                    onClick = onShareEpisode
+                )
+                Divider(
+                    color = Color.LightGray.copy(0.3f),
+                    thickness = 1.dp,
+                )
+                ShareButton(
+                    label = IR.string.podcast_share_current_position,
+                    onClick = onShareCurrentPosition
+                )
+            }
+            if (podcast != null && episode is BaseEpisode && episode.isDownloaded) {
+                ShareButton(label = IR.string.podcast_share_open_file_in, onClick = onShareOpenFileIn)
             }
-            close()
         }
-
-        binding.sharePodcast.isVisible = podcast != null
-        binding.shareEpisode.isVisible = episode != null
-        binding.shareCurrentPosition.isVisible = episode != null
-        binding.openFileIn.isVisible = episode != null && episode.isDownloaded
-
-        return binding.root
     }
+}
 
-    @Suppress("DEPRECATION")
-    override fun onActivityCreated(savedInstanceState: Bundle?) {
-        super.onActivityCreated(savedInstanceState)
+@Composable
+private fun ShareButton(
+    label: Int,
+    onClick: () -> Unit,
+    modifier: Modifier = Modifier
+) {
+    Box(
+        modifier = modifier
+            .fillMaxWidth()
+            .clickable(
+                interactionSource = remember { MutableInteractionSource() },
+                indication = rememberRipple(color = Color.LightGray)
```

Comment:
We should consider to use `MaterialTheme` outline color as divider colours rather than hardcoding them. Wdyt?